### DISCUSSION TITLE
Strange screen after creating helper?

### DISCUSSION THREAD

**Discussion by lindhe** [home-assistant/frontend#21044] [Configuration] [answered]
After adding a threshold sensor helper, I get presented with a screen saying "We found the following devices":

![Screenshot_20240609_034121_Home Assistant](https://github.com/home-assistant/frontend/assets/7773090/d70e58cd-915a-4180-a865-a6d4e4400bf9)

I think this is strange for a number of reasons:

1. I just added the sensor device to my helper, why inform me about it?
2. Why present me with a list when I can only ever add one sensor to this helper? 
3. Why do I get the option to edit the device's location? Feels like the wrong time and place to do that. 

Do you agree that this is strange? Should I report it as a bug?

**Comment by karwosts** [comment, marked as answer]
I agree seems weird, I also see it. 

I would suggest file a core issue for threshold integration. I believe these form flows stem from there.

**Reply by lindhe** [reply]
Thanks for the input. Created issue https://github.com/home-assistant/core/issues/119258
